ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -1926,6 +1926,47 @@ public void shouldIgnoreLegacyDeleteTopicPartOfDropCommand() {
     verifyNoMoreInteractions(schemaRegistryClient);
   }
 
+  @Test
```

Comment:
It seems like we should test some of the logic for waiting to get past the appropriate offsets.